product_specifications/SDerr4/support_doc.md

```diff
@@ -0,0 +1,77 @@
+
+# API Versioning at DataCite
+
+API versioning allows Datacite to continuously evolve the platform while offering third-party developers a predictable path for feature upgrades and deprecations.
+
+To ensure you always know about upcoming API changes, follow the Changelog and always keep your contact information up to date.
+
+## Versioned vs. unversioned APIs
+
+Versioned APIs	
+
+- REST API
+
+Unversioned APIs
+
+- GraphQL API
+- OAI
+- MDS
+- EZID
+
+## Release schedule
+
+Datacite releases a new API version only when is deemed strictly necessary.
+
+Each stable version is supported for a minimum of 12 months. When a new stable version is introduced and contains changes that affect your app, you have 18 months to test and migrate your app to the new version before support for the previous version is set for deprecation. All deprecation introduction would be announced in the DataCite Newsletter and/or DataCite Open Hours.
+We strongly recommend updating your apps to call the latest stable API version. 
+If your request doesn't include a version, then the API also defaults to the oldest supported stable version. 
+
+## Calling an API version
+
+Datacite API versions are explicitly declared in the URL that your app calls:
+
+REST DOIs API URLs: /{ version }/dois/{ id }
+
+For example, the following URLs call version 3:
+
+REST DOIs API URL: /v3/dois/10.2323/102
+REST DOIs (collection) API URL: /v3/dois
+
+
+There are several supported versions of the APIs available, and you specify the version that you want to use by substituting the version name in the URL. There are three types of API versions: `stable`, `release candidate`, and `unsupported`.
```

Comment:
Where does the 'deprecated' fit in?

Reply:
that would be `unsupported` maybe I can normalize the language.